Lib/test/test_parser.py

```diff
@@ -987,5 +988,13 @@ def test_two_args_to_expr(self):
         with self.assertRaises(TypeError):
             parser.expr("a", "b")
 
+
+class TestDeprecation(unittest.TestCase):
+    def test_deprecation_message(self):
+        code = "def f():\n  import parser\n\nf()"
+        rc, out, err = assert_python_ok('-c', code)
+        self.assertIn(b'<string>:2: DeprecationWarning', err)
```

Comment:
Can you use support.check_warnings() with support.import_fresh_module() to check the warning? I don't think so since we cannot reload a C extension twice.

Reply:
Yeah, I don't think it works because C extension module initialization code can only be run once.